### PR TITLE
[Triage Metadata] Inline bug icons for subtests

### LINKED ISSUE #1899: Make it easier to see which failures have been triaged
Currently, no bug links are shown by default, but enabling "Show metadata Information on the wpt.fyi result page." in /flags shows a list at the end.

In a directory like [css/css-images/](https://wpt.fyi/results/css/css-images?label=master&label=experimental&aligned) it's not very easy to figure out which failures have already been triaged. I ended up triaging one test that I didn't know was already triaged in https://github.com/web-platform-tests/wpt-metadata/pull/160.

The UX of this might be tricky, it will easily become cluttered, but something like a bug icon next to the test name or in the individual cells would make it easier to tell what's wrong. Maybe the space that this occupies could also become the thing you click to link new information.

### PR DESCRIPTION
**Pull request by KyleJu**
A follow-up for #1982 and fix #1899.

### CODE REVIEW

**Comment by Hexcles** [comment]
Web component tests failed?

**Comment by Hexcles** [comment]
The UI is a bit misaligned...
image

File: webapp/components/test-file-results-table.js [hunk 1]
```diff
@@ -412,6 +419,28 @@ class TestFileResultsTable extends WPTFlags(Pluralizer(AmendMetadataMixin(WPTCol
...
   }
+
+  shouldDisplayMetadata(index, subtestname, metadataMap) {
+    if (!metadataMap) {
+      return false;
+    }
+    return this.displayMetadata && this.getMetadataUrl(index, subtestname, metadataMap) !== '';
+  }
+
+  getMetadataUrl(index, subtestname, metadataMap) {
```
Comment (Hexcles) [inline webapp/components/test-file-results-table.js:430]: Is this copied from `wpt-results.js`? Can we share them?

**Comment by KyleJu** [comment]
> The UI is a bit misaligned...
> image

Duration row is not supposed to display bug icons. Just fixed it!

File: webapp/components/test-file-results-table.js (hunk 1 above)
Comment (KyleJu) [reply webapp/components/test-file-results-table.js]: It is a bit different. `metadataMap` is a nested map with a format {`key1`: {`key2`: url}} where `key1` is the test name and `key2` is a subtest name or `/`. So the way it retrieves the URL is different between wpt-result.js and test-file-results-table.js.

**Pull request merged**